### PR TITLE
{devel}[intel/2017a] Boost v1.65.1

### PR DESCRIPTION
**Pull request by damianam**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Boost/Boost-1.65.1-intel-2017a-Python-3.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://sourceforge.net/projects/%(namelower)s/files/%(namelower)s/%(version)s']
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+checksums = ['a13de2c8fbad635e6ba9c8f8714a0e6b4264b60a29b964b940a22554705b6b60']
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('Python', '3.6.3'),
```
Comment (verdurin) [inline easybuild/easyconfigs/b/Boost/Boost-1.65.1-intel-2017a-Python-3.6.3.eb]: @damianam I think this dep is missing?

Comment (boegel) [reply easybuild/easyconfigs/b/Boost/Boost-1.65.1-intel-2017a-Python-3.6.3.eb]: Just bumping to `intel/2017b` would fix this...

Comment (damianam) [reply easybuild/easyconfigs/b/Boost/Boost-1.65.1-intel-2017a-Python-3.6.3.eb]: Argh, damnit , I thought I was using `intel/2017b`. Bumping it.........

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2404.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/602de9e5f3649ed55bc2fa5e0fcca9c2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2029.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/97448f31822a77fa7d08c3f92e64da53 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @damianam!

**Pull request merged**